lib/ansible/modules/windows/win_domain_controller.ps1

```diff
@@ -196,6 +196,7 @@ Try {
                 $install_params = @{
                     DomainName = $dns_domain_name
                     Credential = $domain_admin_cred
+                    LogPath = $log_path
```

Comment:
should it even be added to $install_params if $log_path is null?

Reply:
if the param is `null` it ignores it

Reply:
I would still only add it if it was defined like we do with the others, who knows if that behaviour ever changes.